diff --git a/rtslib/fabric.py b/rtslib/fabric.py
index eb8f115..c408661 100644
--- a/rtslib/fabric.py
+++ b/rtslib/fabric.py
@@ -519,14 +519,14 @@ fabric_modules = {
     "srpt": SRPTFabricModule,
     "iscsi": ISCSIFabricModule,
     "loopback": LoopbackFabricModule,
-    "qla2xxx": Qla2xxxFabricModule,
-    "efct": EfctFabricModule,
-    "sbp": SBPFabricModule,
-    "tcm_fc": FCoEFabricModule,
+#    "qla2xxx": Qla2xxxFabricModule,
+#    "efct": EfctFabricModule,
+#    "sbp": SBPFabricModule,
+#    "tcm_fc": FCoEFabricModule,
 #    "usb_gadget": USBGadgetFabricModule, # very rare, don't show
-    "vhost": VhostFabricModule,
-    "xen-pvscsi": XenPvScsiFabricModule,
-    "ibmvscsis": IbmvscsisFabricModule,
+#    "vhost": VhostFabricModule,
+#    "xen-pvscsi": XenPvScsiFabricModule,
+#    "ibmvscsis": IbmvscsisFabricModule,
     }
 
 #
